tests/meta/orders/test_meta_orders_status.py

```diff
@@ -215,7 +215,7 @@ def test_store_microbial_samples(orders_api, base_store, microbial_status_data):
 
     # THEN it should store the samples and the used previously unknown organisms
     assert new_samples
-    assert base_store.families().count() == 0
+    assert base_store.families().count() == 1
```

Comment:
Could you add "THEN" statements for all asserts? It is hard to understand why the test has changed without any explanation

Reply:
fixed